concent_api/concent_api/settings/development.py

```diff
@@ -18,3 +18,5 @@
     "gatekeeper",
     "admin-panel",
 ]
+
+STORAGE_CLUSTER_ADDRESS = 'http://storage.concent.golem.network/'
```

Comment:
Set the default to `'http://devel.concent.golem.network/'`. storage.concent.golem.network Does not exist yet.

Reply:
Done